packages/preview-button/admin/src/components/Content/index.jsx

```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Markdown } from '../Markdown';
+
+export const Content = ({ data, locale, priceData, priceZeroLabel = '€ 0,00', title }) => {
```

Comment:
De currency is niet per se euro toch? Misschien moeten we hier de [Currency sign](https://en.wikipedia.org/wiki/Currency_sign_(generic)) gebruiken:
```suggestion
export const Content = ({ data, locale, priceData, priceZeroLabel = '¤ 0,00', title }) => {
```